lib/statistics.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class Statistics
+  def self.active_users
+    {
+      :last_day => User.where("last_seen_at > ?", 1.days.ago).count,
+      "7_days" => User.where("last_seen_at > ?", 7.days.ago).count,
+      "30_days" => User.where("last_seen_at > ?", 30.days.ago).count,
+    }
+  end
+
+  def self.likes
+    {
+      :last_day =>
+        UserAction.where(action_type: UserAction::LIKE).where("created_at > ?", 1.days.ago).count,
+      "7_days" =>
+        UserAction.where(action_type: UserAction::LIKE).where("created_at > ?", 7.days.ago).count,
+      "30_days" =>
+        UserAction.where(action_type: UserAction::LIKE).where("created_at > ?", 30.days.ago).count,
+      :count => UserAction.where(action_type: UserAction::LIKE).count,
+    }
+  end
+
+  def self.posts
+    {
+      :last_day => Post.where("created_at > ?", 1.days.ago).count,
+      "7_days" => Post.where("created_at > ?", 7.days.ago).count,
+      "30_days" => Post.where("created_at > ?", 30.days.ago).count,
+      :count => Post.count,
+    }
```

Comment:
we mix symbol keys and string keys in the whole class, we should avoid this

Reply:
Also we should use the more modern syntax: `count: Post.count`

Reply:
Hm, I copied this pattern from another similar file we have https://github.com/discourse/discourse/blob/main/plugins/chat/lib/chat/statistics.rb. I'm guessing that was written this way because names of some keys start from numbers. But you're right, and actually we can write it like this (keys that starts from numbers are in quotes, but they are still symbols):

```ruby
      last_day: Post.where("created_at > ?", 1.days.ago).count,
      "7_days: Post.where("created_at > ?", 7.days.ago).count,
      "30_days": Post.where("created_at > ?", 30.days.ago).count,
      count: Post.count,
```